requirements.txt

```diff
@@ -1,3 +1,4 @@
+vmprof==0.4.12
```

Comment:
Add this here, because github won't let me comment on submodule updates. Those are likely mistakes that you ought to remove, or is there something I should look at changed in the CPython test suites by you?